executor/prepared_test.go

```diff
@@ -25,14 +24,13 @@ import (
 )
 
 func (s *testSuite) TestPrepared(c *C) {
-	cfg := config.GetGlobalConfig()
-	orgEnable := cfg.PreparedPlanCache.Enabled
-	orgCapacity := cfg.PreparedPlanCache.Capacity
+	orgEnable := plan.PreparedPlanCacheEnabled
+	orgCapacity := plan.PreparedPlanCacheCapacity
 	flags := []bool{false, true}
 	ctx := context.Background()
 	for _, flag := range flags {
-		cfg.PreparedPlanCache.Enabled = flag
-		cfg.PreparedPlanCache.Capacity = 100
+		plan.PreparedPlanCacheEnabled = flag
```

Comment:
Will it cause data race during test? Maybe other goroutine is changing this global variable concurrently during test.

Reply:
Seem `cfg` and `plan` are all global variable? So it's safe to do this?